### PR TITLE
Fix docs typos

### PR DESCRIPTION
**Pull request by mariosasko**
This PR fixes few typos in the docs and the error in the code snippet in the set_format section in docs/source/torch_tensorflow.rst. `torch.utils.data.Dataloader` expects padded batches so it throws an error due to not being able to stack the unpadded tensors. If we follow the Quick tour from the docs where they add the `truncation=True, padding='max_length'` arguments to the tokenizer before passing data to Dataloader, we can easily fix the issue.

### CODE REVIEW

**Review by lhoestq — APPROVED** [review]
Good catch ! thank you for fixing this :)
